all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -40,29 +40,26 @@ metadata:
     name: jaeger-infra
     jaeger-infra: jaeger-template-all-in-one
 objects:
-- apiVersion: v1
-  kind: DeploymentConfig
+- apiVersion: extensions/v1beta1
```

Comment:
Perhaps. In any case, I'd not mention any versions here, as it has the potential of becoming a rabbit hole. Once this gets really supported, then we should certainly determine the minimum versions.